src/api/api.ts

```diff
@@ -775,47 +775,215 @@ export interface CellUpdate {
+     * @memberof CheckBase
+     */
+    orgID: string;
+    /**
+     * The ID of the authorization used to create this check.
+     * @type {string}
+     * @memberof CheckBase
+     */
+    authorizationID?: string;
     /**
      * 
-     * @type {Array<Check>}
-     * @memberof Check
+     * @type {Date}
+     * @memberof CheckBase
      */
-    checks?: Array<Check>;
+    createdAt?: Date;
+    /**
+     * 
+     * @type {Date}
+     * @memberof CheckBase
+     */
+    updatedAt?: Date;
     /**
      * 
+     * @type {DashboardQuery}
+     * @memberof CheckBase
+     */
+    query: DashboardQuery;
+    /**
+     * The status of the check task.
+     * @type {string}
+     * @memberof CheckBase
+     */
+    status?: CheckBase.StatusEnum;
+    /**
+     * Check repetition interval
+     * @type {string}
+     * @memberof CheckBase
+     */
+    every?: string;
+    /**
+     * Duration to delay after the schedule, before executing check.
+     * @type {string}
+     * @memberof CheckBase
+     */
+    offset?: string;
+    /**
+     * Check repetition interval in the form '* * * * * *';
+     * @type {string}
+     * @memberof CheckBase
+     */
+    cron?: string;
+    /**
+     * tags to write to each status
+     * @type {Array<CheckBaseTags>}
+     * @memberof CheckBase
+     */
+    tags?: Array<CheckBaseTags>;
+    /**
+     * template that is used to generate and write a status message
      * @type {string}
-     * @memberof Check
+     * @memberof CheckBase
+     */
+    statusMessageTemplate?: string;
+    /**
+     * 
+     * @type {Array<Label>}
+     * @memberof CheckBase
      */
-    status: Check.StatusEnum;
+    labels?: Array<Label>;
 }
 
 /**
  * @export
- * @namespace Check
+ * @namespace CheckBase
  */
-export namespace Check {
+export namespace CheckBase {
     /**
      * @export
      * @enum {string}
      */
     export enum StatusEnum {
-        Pass = 'pass',
-        Fail = 'fail'
+        Active = 'active',
+        Inactive = 'inactive'
     }
 }
 
+/**
+ * 
+ * @export
+ * @interface CheckBaseTags
+ */
+export interface CheckBaseTags {
+    /**
+     * 
+     * @type {string}
+     * @memberof CheckBaseTags
+     */
+    key?: string;
+    /**
+     * 
+     * @type {string}
+     * @memberof CheckBaseTags
+     */
+    value?: string;
+}
+
+/**
+ * the state to record if check matches a criteria
+ * @export
+ * @enum {string}
+ */
+export enum CheckStatusLevel {
+    UNKNOWN = 'UNKNOWN',
+    OK = 'OK',
+    INFO = 'INFO',
+    CRIT = 'CRIT',
+    WARN = 'WARN'
+}
+
+/**
+ * 
+ * @export
+ * @interface CheckViewProperties
+ */
+export interface CheckViewProperties {
+    /**
+     * 
+     * @type {string}
+     * @memberof CheckViewProperties
+     */
+    type?: CheckViewProperties.TypeEnum;
+    /**
+     * 
+     * @type {string}
+     * @memberof CheckViewProperties
+     */
+    checkID?: string;
+    /**
+     * 
+     * @type {Check}
+     * @memberof CheckViewProperties
+     */
+    check?: Check;
```

Comment:
checkID is part of the CheckView data type, but the separate check data type is going to be bundled in by the API, so that we don't have to make an extra request for the check. FE can only update checkID via `/view` endpoint and can only update check parameters via `/checks` endpoint. I think this is fine.